setup.py

```diff
@@ -1,44 +1,38 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
-import os
+import pathlib
 
+from pkg_resources import parse_requirements
 from setuptools import setup, find_packages
 
+# Repository root directory for use with reading files
+directory = pathlib.Path(__file__).parent
 
-# Utility function to read the README file.
-# Used for the long_description.  It's nice, because now 1) we have a top level
-# README file and 2) it's easier to type in the README file than to put a raw
-# string in below ...
-def read(fname):
-    return open(os.path.join(os.path.dirname(__file__), fname)).read()
-
-
-install_requires_replacements = {}
-
-install_requirements = list(set(
-    install_requires_replacements.get(requirement.strip(), requirement.strip())
-    for requirement in open('requirements.txt')
-    if not requirement.lstrip().startswith('#')
-))
+# Load install_requires from requirements.txt.
+# https://stackoverflow.com/a/59971236/4651668
+requirements = directory.joinpath('requirements.txt').read_text()
+requirements = [str(r) for r in parse_requirements(requirements)]
```

Comment:
running this on the text from https://github.com/rotki/rotki/blob/9a3ea165b851414059f271f192b4cb190c8a3c70/requirements.txt, I get:

```
['gevent==1.5a2', 'greenlet==0.4.16', 'web3==5.11.1', 'pysqlcipher3==1.0.3', 'requests==2.20.0', 'pycryptodome==3.6.6', 'pycryptodomex==3.6.6', 'coincurve==13.0.0', 'typing-extensions==3.7.4.2', 'base58check==1.0.2', 'bech32==1.2.0', 'gql==2.0.0', 'flask-cors==3.0.8', 'flask-restful==0.3.7', 'flask==1.1.1', 'marshmallow==3.2.1', 'webargs==6.0.0', 'setuptools==41.0.1', 'eth-utils==1.8.4', 'rlp==1.2.0']
```

Also confirmed environment markers, as suggested at https://github.com/rotki/rotki/pull/1246#discussion_r507195675, don't get stripped by parse_requirements.

Reply:
Thanks, this actually looks cleaner.